TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/RobotTools/TelemetryDump.java

```diff
@@ -1,48 +1,34 @@
 package org.firstinspires.ftc.teamcode.rework.RobotTools;
 
 import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.teamcode.rework.ModuleTools.TelemetryProvider;
 
+import java.util.ArrayList;
 import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
 
 public class TelemetryDump {
     Telemetry telemetry;
-
-    LinkedHashMap<String,String> opModeData;
-    LinkedHashMap<String,String> executorModuleData;
+    private static List<TelemetryProvider> providers = new ArrayList<>();
```

Comment:
Why is this a List and not an ArrayList? Just curious

Reply:
Also probably better to initialize it in the constructor like what was done with the LInkedHashMaps

Reply:
its a list and not an arraylist just because its a stupid habit; will fic